tests/ext/aws/test_publisher.py

```diff
@@ -18,8 +18,8 @@ def test_sqs_publish(mock_boto_client_sqs):
             QueueUrl=queue_url)
```

Comment:
These guys are still `mock.Mock`. I didn't change `sqs_publish()` and `sns_publish()` functions to `aiobotocore` because they are being used by the cli. Thus making them `async` would break it (the cli) and wouldn't be easy to fix it.